### DISCUSSION TITLE
Mirth 4.4.2 not able to accept messages if listening on specific interface

### DISCUSSION THREAD

**Discussion by ramsoftmarc** [nextgenhealthcare/connect#6041] [Q&A] [answered]
This wasn't an issue before, but since the update, we've had to select "All interfaces".  Is this by design?

**Comment by jonbartels** [comment, marked as answer]
https://github.com/nextgenhealthcare/connect/wiki/4.3.0---What's-New#the-tcp-listener-now-correctly-binds-to-the-specified-local-address

https://github.com/nextgenhealthcare/connect/issues/5161

**Reply by ramsoftmarc** [reply]
thanks!
